images/dptp-3312-hypershift-leaks-cleaner/cleanup-hypershift-leaks.sh

```diff
@@ -1,19 +1,37 @@
 #!/bin/bash
 NOW=$(date +%s)
 echo Cleanning up resources older than $(date -u -d @$NOW +%Y-%m-%dT%H:%M:%SZ)
-AWS_CRED_PATH=${AWS_CRED_PATH:-/token/.awscred}
-echo Using AWS_CRED_PATH=${AWS_CRED_PATH}
+
+# int his script we are assuming that the credentials are mounted according to their cluster-profile
+# e.g. cluster-profile=aws, AWS_CONFIG_FILE=/aws/token/.awscred
+# e.g. cluster-profile=aws2, AWS_CONFIG_FILE=/aws2/token/.awscred
+updateCredentialsEnv() {
+    cluster_profile=`oc -n clusters get hostedclusters/$1 -o json | jq -r '.metadata.annotatios."cluster-profile"'`
+    if [ $cluster_profile == null ]; then 
+        echo "error: No cluster profile found for $1."
+        return
+    fi
+    if ! [ -f "$cluster_profile/token" ]; then
+        echo "error: Cluster profile secret: $cluster_profile not present."
+        return
+    fi
+    AWS_CONFIG_FILE=${AWS_CONFIG_FILE:-/$cluster_profile/token/.awscred}
```

Comment:
I don't think we need to support overriding `AWS_CONFIG_FILE` in this case - it is a dynamic value.